src/components/PersonalTweets/PersonalTweet.css

```
@@ -0,0 +1,27 @@
+.personalTweets {
+  padding-top: 1%;
+  padding-bottom: 1%;
+  padding-left: 1%;
+  border-bottom: 1px solid black;
+  display: grid;
+  grid-template-columns: 75px 1fr;
+}
+
+
+img{
```

```suggestion
img {
```
Try to keep syntax spacing consistent across files.